packages/easy-coding-standard/src/Console/Style/EasyCodingStandardStyle.php

```diff
@@ -54,7 +55,7 @@ public function printErrors(array $codingStandardErrors): void
     public function enableDebugProgressBar(): void
     {
         $privatesAccessor = new PrivatesAccessor();
-        $progressBar = $privatesAccessor->getPrivateProperty($this, 'progressBar');
+        $progressBar = $privatesAccessor->getPrivatePropertyOfClass($this, 'progressBar', ProgressBar::class);
```

Comment:
as we are using real class-references in the code, the code now tells this dependency.

this was not visible before this PR.